client/src/features/calendar/MonthAndYear.js

```diff
@@ -11,7 +11,7 @@ const MonthAndYear = ({
       <div className="flex">
         <button onClick={handlePreviousMonth}>
           <svg
-            className="w-6 h-6"
+            className="w-6 h-6 max-[440px]:w-4 max-[440px]:h-4"
```

Comment:
I haven't played around with the breakpoints myself at this point, but do we think these `440px` and `380px` breakpoints should be [refactored into the Tailwind config](https://tailwindcss.com/docs/responsive-design#customizing-your-theme) for easier reuse (or if they need to be updated later) or are they more of a one-off kind of thing?

Reply:
> I haven't played around with the breakpoints myself at this point, but do we think these `440px` and `380px` breakpoints should be [refactored into the Tailwind config](https://tailwindcss.com/docs/responsive-design#customizing-your-theme) for easier reuse (or if they need to be updated later) or are they more of a one-off kind of thing?

I think tailwind config is a better way to go in the long run.

Reply:
Hey, sorry I've only just now seen this! I agree that the tailwind config is the better way to go, in case there are more cases like this in the future if other features are added to the app. I'd be happy to work on the mobile-responsiveness of the modal as well if needed, either in this PR or a different one!

Reply:
I'd prefer not to add it to this PR because there are a couple ways we could solve the modal that'd I'd like to investigate before we start working on it.

Reply:
This should be good then, so I'll go ahead and approve as well. I'll make sure to include some language in the tech spec I'll be writing later this week for the modal/mobile design and specifically for the tailwind breakpoints.